sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -272,6 +272,31 @@ const BuildingDetails = ({
           />
         </GridCell>
       </GridSection>
+      <GridSection columns={3}>
+        <GridCell>
+          <Field
```

Comment:
This should definitely be a dropdown with counties IMO -- see states for an example of how to do this

Reply:
not necessary in this CL but we should also add validation on the backend to make sure that county is a valid field before saving (and potentially even make this field required)

Reply:
Great! That's what I was thinking as well. Let's do that in the next PR, as you suggest. Thanks for the reinforcement.

Reply:
I obviously ended up doing it in this PR.